### DISCUSSION TITLE
How to downgrade Vite version.

### DISCUSSION THREAD

**Discussion by vinoddotcom** [nuxt/nuxt#28275] [Questions] [answered]
@huang-julien

I'm encountering errors with vite@5.3.4 during the build process and need to downgrade to vite@5.3.1. However, I'm unsure where to specify the Vite version in the project configuration. Could you guide me on how to do this?

**Comment by TheAlexLichter** [comment, marked as answer]
Use your package managers override or resolution field to do so. [Example of that in pnpm](https://pnpm.io/package_json#pnpmoverrides)

**Comment by vinoddotcom** [comment]
Thank you, @manniL. I appreciate your answer. However, I still have an issue with my build. Please refer to https://github.com/nuxt/nuxt/issues/28293.
